src/layout/Navbar/Navbar.tsx

```diff
@@ -1,19 +1,23 @@
-import { useContext, useEffect, useState } from "react";
+import { useEffect, useState } from "react";
 import { NavLink } from "react-router-dom";
-import { fetchUserData } from "services/api";
-import { UserContext } from "context/UserProvider";
 import styles from "./navbar.module.css";
 import Profile from "./Profile/Profile";
-
+import { GetUserResponse } from "types/apis";
 import logoImage from "assets/images/logo.svg";
+import { fetchGetSampleUsers } from "utils/hooks/useGetSampleData";
 
 function Navbar() {
-  const { id } = useContext(UserContext);
-  const [user, setUser] = useState(null);
+  const [user, setUser] = useState<GetUserResponse>();
+
+  const setLocalStorage = (key: string, val: number): void =>
+    localStorage.setItem(key, JSON.stringify(val));
```

Comment:
중괄호는 꼭 써주면 좋을 것 같아요~
```suggestion
  const setLocalStorage = (key: string, val: number): void => {
    localStorage.setItem(key, JSON.stringify(val));
}
```

Reply:
이렇게 간단한 함수의 경우에는 리턴타입은 따로 잡아주지 않아도 괜찮을 것 같아요~
```suggestion
  const setLocalStorage = (key: string, val: number) =>
    localStorage.setItem(key, JSON.stringify(val));
```

Reply:
이 함수는 utils로 만들어놓고 사용해줘도 괜찮을 것 같은데요~
한줄짜리 함수를 어떤식으로 추상화를 할지에 대해서 고민해보시면 좋을 것 같아요!